hack/generator/pkg/codegen/pipeline_remove_type_aliases.go

```diff
@@ -49,7 +50,7 @@ func removeTypeAliases() PipelineStage {
 func resolveTypeName(visitor *astmodel.TypeVisitor, name astmodel.TypeName, types astmodel.Types) (astmodel.Type, error) {
 	def, ok := types[name]
 	if !ok {
-		panic(fmt.Sprintf("Couldn't find type for type name %s", name))
+		return nil, errors.Errorf("Couldn't find definition for type name %s", name)
```

Comment:
minor: lowercase first letter of error because Go hates us

Reply:
Fixed.